src/main/java/frc/robot/subsystems/Transfer.java

```diff
@@ -15,9 +15,9 @@
 public class Transfer extends MotoredGenericSubsystem {
```

Comment:
I've spoken to Sitton and we're probably not going to be using a limelight, so we need to decide and change the class accordingly.